components/modal/modal.js

```diff
@@ -8,9 +8,15 @@ import { getClassNamesWithMods } from '../_helpers';
  * Modal component
  */
 class Modal extends Component {
-  state = {
-    isOpen: false,
-  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isActive: props.active || false,
```

Comment:
Since you added this (nice one 👍 ), do you still need this one: https://github.com/Travix-International/travix-ui-kit/pull/115/files#diff-aeb4723b4b18ec633e3c8bab2ea276e6R25 ?